### PR TITLE
Adding docs for the new restrict access to Dashboards and Monitors functionality

### PR DESCRIPTION
**Pull request by apigirl**
### What does this PR do?
Adding docs for the new restrict access to Dashboards and Monitors functionality

- https://docs-staging.datadoghq.com/kaylyn/restrict-monitor/account_management/rbac/#restrict-access-to-dashboards-and-monitors
- https://docs-staging.datadoghq.com/kaylyn/restrict-monitor/dashboards/#restrict-access
- https://docs-staging.datadoghq.com/kaylyn/restrict-monitor/getting_started/application/monitors/#restrict-access

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by kayayarai** [comment]
This looks fine except for the empty `static/images/account_management/saml/google_idp_tile_sml.png` file included?

File: content/en/account_management/rbac/_index.md [hunk 1]
```diff
@@ -134,6 +134,10 @@ Find an example of how to delete a Role in the [Datadog Create Role API document
 {{% /tab %}}
 {{< /tabs >}}
 
+## Restrict access to Dashboards and Monitors
```
Comment (cswatt) [inline content/en/account_management/rbac/_index.md:137]:
```suggestion
## Restrict access to dashboards and monitors
```

File: content/en/account_management/rbac/_index.md [hunk 2]
```diff
@@ -134,6 +134,10 @@ Find an example of how to delete a Role in the [Datadog Create Role API document
...
+
+Once you have RBAC roles set up, you can restrict access to Dashboards and Monitors by user role. For more information, see the [Dashboard permissions docs][8], and the [Monitors permissions docs][9].
```
Comment (cswatt) [inline content/en/account_management/rbac/_index.md:139]:
```suggestion
Once you have RBAC roles set up, you can restrict access to dashboards and monitors by user role. For more information, see the [Dashboard Permissions docs][8] and the [Monitors Permissions docs][9].
```

**Review by cswatt — CHANGES_REQUESTED** [review]
small edits

**Review by kayayarai — APPROVED** [review]

**Pull request merged**